src/api/lib/rabbitmq_bus.rb

```diff
@@ -3,8 +3,7 @@ def self.publish(event_queue_name, event_payload)
     return unless CONFIG['amqp_options']
     start_connection
 
-    queue = $rabbitmq_channel.queue(event_queue_name, CONFIG['amqp_queue_options'].try(:with_indifferent_access) || {})
-    $rabbitmq_exchange.publish(event_payload, routing_key: queue.name)
+    $rabbitmq_exchange.publish(event_payload, routing_key: event_queue_name)
```

Comment:
Actually self.class.message_bus_queue is already a misnomer. We don't want different queues per event type, just different routing keys.

Reply:
I know that is a PITA but I would prefer to change the name if it doesn't really make sense... something like ```self.class.message_bus_route``` is better??

Reply:
And also then changing ```event_queue_name``` to ```event_route``` ??

Reply:
Your suggestion would be better than what we have. To quote from the net: "A message goes to the queue(s) whose binding key exactly matches the routing key of the message.". What we are passing here is that routing key - so possibly we would be most correct if we called it event_routing_key